docs/wiki/The library - Module design.md

```diff
@@ -655,13 +659,27 @@ In addition, they follow these file-type-specific guidelines:
 
 # Telemetry
 
+## Overview 
+
+Microsoft uses the approach detailed in this section to identify the deployments of the Bicep and ARM JSON templates of the CARML library. Microsoft collects this information to provide the best experiences with their products and to operate their business. Telemetry data is captured through the built-in mechanisms of the Azure platform; therefore, it never leaves the platform, providing only Microsoft with access. Deployments are identified through a specific GUID (Globally Unique ID) representing the code is originated in the CARML library. The data is collected and governed by Microsoft's privacy policies, located at the [Trust Center](https://www.microsoft.com/en-us/trust-center).
```

Comment:
"Deployments are identified through a specific GUID (Globally Unique ID), indicating that the code originated from the CARML library"

Reply:
Updated as suggested.